clients/src/main/java/org/apache/kafka/common/quota/ClientQuotaEntity.java

```diff
@@ -32,6 +32,7 @@
      */
     public static final String USER = "user";
     public static final String CLIENT_ID = "client-id";
+    public static final String IP = "IP";
```

Comment:
Consider making lower-case for consistency, but otherwise not a big issue, as I see that's how it was proposed in the KIP.

Reply:
I have no objection to editing the KIP to changing this (and other uses of IP) to lower case, I just made it upper case because the original iteration of IP quotas in KIP-612 had IP in upper-case.

@apovzner @dajac for any thoughts on upper vs lower.
FWIW, the ConfigCommand help output is better ordered when IP is in lower-case due to how it sorts flags.

Reply:
as I mentioned in my other comment, agreed with using lower-case.